### ISSUE TITLE
Validation for aria-dropeffect fails valid combinations

### ISSUE THREAD

**Issue by dd8** [validator/validator#915]
https://www.w3.org/TR/wai-aria-1.1/#aria-dropeffect

I can't see anything in the normative text that prohibits certain combinations of drag-and-drop operations. In particular the copy+move combination is flagged as invalid, but is very common in file drag-and-drops:

```
aria-dropeffect='copy move'
``` 

aria-dropeffect is marked as deprecated in ARIA 1.1 (and probably has close to zero support) so simplest fix might be removing validation of aria-dropeffect values if this is tricky to express in RNC

**Comment by sideshowbarker** [comment]
> aria-dropeffect is marked as deprecated in ARIA 1.1 (and probably has close to zero support) so simplest fix might be removing validation of aria-dropeffect values if this is tricky to express in RNC

Yeah, given that I went ahead and changed it to allow any value

**Comment by dd8** [comment]
Sorry, discovered  some ACT Rules test cases for both valid and invalid aria-dropeffect values:
https://act-rules.github.io/rules/6a7281#passed-example-9
https://act-rules.github.io/rules/6a7281#failed-example-8

I submitted a PR with an updated fix
